src/components/Form/Search.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import T from 'prop-types'
+import { classNames, randomId } from '../../utils/'
+
+const initial = 'c-search'
+
+const Search = ({ className, small, type = 'search', placeholder = 'Search....', submitPosition, buttonTitle = 'Submit', ...rest }) => (
+  <form role="search" action="" method="" className={classNames(initial, submitPosition ? `${initial}--submit-${submitPosition}` : '', className)}>
+    <button title={buttonTitle} type="submit" className={classNames('c-search__submit c-btn c-btn--alpha')} />
+    <input
+      id={randomId()}
+      type={type}
```

Comment:
Shouldn't be a default prop.  Should be a static value of 'search'

Reply:
done